lib/game.js

```diff
@@ -13,15 +13,16 @@ class Game {
 
 	start(){
 		var _this = this;
+    var velocityX = 0;
 		window.requestAnimationFrame(function gameLoop(){
-			var position = _this.backgroundPosition-=0.25;
-			var offset = _this.backgroundOffset+=0.25;
-			_this.ctx.clearRect(0, 0, _this.canvas.width, _this.canvas.height)
-			_this.ctx.save();
-			_this.ctx.translate(_this.backgroundOffset, 0);
-			_this.ctx.drawImage(_this.background, position, 0, 900, 400)
-			_this.ctx.drawImage(_this.background, (position + _this.background.width), 0, 900, 400)
-			requestAnimationFrame(gameLoop);
+      requestAnimationFrame(gameLoop);
+      _this.ctx.clearRect(0, 0, _this.canvas.width, _this.canvas.height)
+			_this.ctx.drawImage(_this.background, velocityX, 0)
+			_this.ctx.drawImage(_this.background, (_this.background.width-Math.abs(velocityX)), 0);
+      if (Math.abs(velocityX) > _this.background.width) {
```

Comment:
line 22 is asking whether the position of the background image's edge has reached the edge of the canvas. The Math.abs bit is allowing for the canvas to move in either direction and continue to scroll. I know we don't need that, but when I took it out, it broke and I was tired and it was 1am, so I left it in. we can probably refactor that if we put our minds to it (after sleep)